torch/_inductor/ir.py

```diff
@@ -7744,8 +7751,7 @@ def add_submodule(self, block, prefix):
         return name
 
     def add_indirect(self, size):
-        name = f"indirect{len(self.indirect_vars)}"
-        var = sympy_index_symbol(name)
+        var = sympy_index_symbol_with_prefix(SymT.INDIRECT, len(self.indirect_vars))
```

Comment:
This is a placeholder variable that is later replaced with `SymT.TMP`. Can you add this to the description of the `SymT` symbol?